logpyle/__init__.py

```diff
@@ -134,7 +137,15 @@ def __call__(self) -> Optional[float]:
 
         This is only called if the invocation interval calls for it.
         """
-        raise NotImplementedError
+        raise NotImplementedError(
+            f"__call__ not implemented for log quantity {self.name}."
+        )
+
+    def set_value(self, value: Any) -> None:
+        """Set the logged quantity value."""
+        raise NotImplementedError(
+            f"set_value not implemented for log quantity {self.name}."
+        )
```

Comment:
Is it avoidable to have `set_value` in the `LogQuantity` interface instead of only in `PushLogQuantity`? Having it breeds an expectation that every `LogQuantity` support it, which isn't right.